src/server/scripts/Battlegrounds/DalaranSewers/arena_dalaran_sewers.cpp

```diff
@@ -259,8 +256,74 @@ class at_ds_pipe_knockback : public AreaTriggerScript
     }
 };
 
+// 96539 - Pipe Flush Knockback Search Trigger
+class spell_arena_dalaran_sewers_pipe_flush_knockback_search_trigger : public AuraScript
+{
+    bool Validate(SpellInfo const* spellInfo) override
+    {
+        return ValidateSpellEffect({
+            { spellInfo->Id, EFFECT_0 },
+            { spellInfo->GetEffect(EFFECT_0).TriggerSpell, EFFECT_0 }
+        }) && ValidateSpellInfo({
+            static_cast<uint32>(sSpellMgr->GetSpellInfo(spellInfo->GetEffect(EFFECT_0).TriggerSpell, spellInfo->Difficulty)->GetEffect(EFFECT_0).CalcValue())
+        });
+    }
+
+    void HandlePeriodicTrigger(AuraEffect const* aurEff)
+    {
+        // The trigger spell is never cast on retail
```

Comment:
we have proof of blizzard's magic hiding start and go packets so there's no trace of the spell happening on sniffs even though its effect happens, might be the case here